pinc/MARCRecord.inc

```diff
@@ -154,26 +155,33 @@ class MARCRecord
 
     public function get_language()
     {
-        $marc_language = "";
         $key = $this->_key_search("008", "");
         if(isset($this->record[$key]))
         {
             $marc_language = substr($this->record[$key][1], 35, 3);
-            $marc_language = convert_short_lang($marc_language);
+            $lang_name = langname_for_langcode3($marc_language);
+            if($lang_name)
+            {
+                return $lang_name;
```

Comment:
Consider keeping `$marc_language` and only having one `return` in the function (and `$marc_literary_form` below). When possible, having one return makes the code flow easier to understand and avoids bugs in the future where the dev is unaware that there are multiple returns (clearly no the case here on these small functions, but a general good rule-of-thumb).

Reply:
Done this.